ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -306,11 +348,27 @@ export default class StatusDropdown extends React.Component {
 
     const dueDateStatus = this.displayDueDateCountdown();
 
-    let dueDateClass = 'due-date-ok';
-    if (dueDateStatus === 'Overdue') {
-      dueDateClass = 'due-date-overdue';
-    } else if (dueDateStatus.startsWith('Hours left:')) {
-      dueDateClass = 'due-date-today';
+    let dueDateClass;
+    if (!alertSummary.first_triaged) {
+      if (dueDateStatus.triage === 'Overdue') {
+        dueDateClass = 'due-date-overdue';
+      } else if (dueDateStatus.triage.endsWith('hours left')) {
+        dueDateClass = 'due-date-today';
+      } else {
+        dueDateClass = 'due-date-ok';
+      }
+    } else {
+      if (alertSummary.bug_number) {
+        dueDateClass = 'due-date-done';
+      } else {
+        if (dueDateStatus.bug === 'Overdue') {
+          dueDateClass = 'due-date-overdue';
+        } else if (dueDateStatus.bug.endsWith('hours left')) {
+          dueDateClass = 'due-date-today';
+        } else {
+          dueDateClass = 'due-date-ok';
+        }
+      }
```

Comment:
This is pretty difficult to follow. Maybe we could simplify it to make it more readable.